test/regression/aarch64/instructions/float.cc

```diff
@@ -1453,6 +1453,37 @@ TEST_P(InstFloat, ucvtf) {
   CHECK_NEON(9, float, {static_cast<float>(UINT64_C(1) << 48), 0.f, 0.f, 0.f});
   CHECK_NEON(10, float, {static_cast<float>(UINT64_MAX), 0.f, 0.f, 0.f});
   CHECK_NEON(11, float, {0.f, 0.f, 0.f, 0.f});
+
+  // 32-bit unsigned fixed-point to float
+  // Numbers have been chosen to have less than 0.0005 fixed-point
+  // representation error to ensure tests pass
+  initialHeapData_.resize(12);
+  heap32 = reinterpret_cast<uint32_t*>(initialHeapData_.data());
+  heap32[0] = 0x000001EE;
```

Comment:
Could you comment on what these values represent in float form, e.g. `heap32[0] = 0x000001EE; // 0.0....`